pages/perfherder.py

```diff
@@ -13,7 +13,6 @@ class PerfherderPage(Base):
 
     def wait_for_page_to_load(self):
         self.wait.until(lambda s: self.is_graph_chooser_displayed)
-        return self
```

Comment:
This one makes sense to keep as we're extending the `Page.wait_for_page_to_load` method from PyPOM, which returns the page object. See http://pypom.readthedocs.io/en/latest/_modules/pypom/page.html#Page.wait_for_page_to_load